cli/logger/color_logger.go

```diff
@@ -9,23 +9,27 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 )
 
+// ColorLoggerFactory implements logger.Factory interface using ColorLogger.
 type ColorLoggerFactory struct {
 	maxLength int
 	tty       bool
 }
 
+// ColorLogger implements logger.Logger interface with color support.
 type ColorLogger struct {
```

Comment:
I think this one could be exported, if not used by a project depending on `libcompose` — it then depends on wether we want it to be exported or not :wink:.